packages/ag-charts-community/src/chart/dom/theme.css

```diff
@@ -0,0 +1,106 @@
+    --ag-charts-border-critical: var(--ag-borders-critical, solid 1px);
+
+    /* Layout */
+    --ag-charts-layer-context-menu: 5;
+    --ag-charts-layer-tooltip: 4;
+    --ag-charts-layer-toolbar: 3;
+    --ag-charts-layer-crosshair: 2;
+    --ag-charts-layer-annotations: 1;
+    --ag-charts-align: center;
+    --ag-charts-justify: center;
+
+    /* Axis Labels */
+    --ag-charts-axis-label-background-color: #474747;
+    --ag-charts-axis-label-color: #ffffff;
+
+    /* Toolbar */
+    --ag-charts-toolbar-foreground-color: var(--ag-header-foreground-color, var(--ag-charts-foreground-color));
+    --ag-charts-toolbar-background-color: var(
+        --ag-header-background-color,
+        color-mix(in srgb, var(--ag-charts-background-color), var(--ag-charts-foreground-color) 2%)
+    );
+    --ag-charts-toolbar-size: 34px;
+    --ag-charts-toolbar-size-small: 24px;
+    --ag-charts-toolbar-border: var(--ag-charts-border-critical, solid 1px) var(--ag-charts-border-color);
+    --ag-charts-toolbar-hover-color: color-mix(
+        in srgb,
+        var(--ag-charts-background-color),
+        var(--ag-charts-active-color) 12%
+    );
+    --ag-charts-toolbar-focus-color: var(--ag-charts-active-color);
+    --ag-charts-toolbar-active-background-color: color-mix(
+        in srgb,
+        var(--ag-charts-background-color),
+        var(--ag-charts-active-color) 24%
+    );
+    --ag-charts-toolbar-active-color: var(--ag-active-color, #5090dc);
+    --ag-charts-toolbar-disabled-foreground-color: var(
+        --ag-disabled-foreground-color,
+        color-mix(in srgb, transparent, var(--ag-charts-toolbar-foreground-color) 50%)
+    );
+    --ag-charts-toolbar-disabled-background-color: color-mix(
+        in srgb,
+        var(--ag-charts-toolbar-background-color),
+        var(--ag-charts-toolbar-foreground-color) 6%
+    );
+    --ag-charts-toolbar-gap: var(--ag-charts-size);
+    --ag-charts-toolbar-font-size: 14px;
+    --ag-charts-toolbar-font-size-small: 12px;
+    --ag-charts-toolbar-button-padding: 6px;
+    --ag-charts-toolbar-button-padding-small: 1px;
+
+    /* Popover */
+    --ag-charts-popover-background-color: #fafafb;
+    --ag-charts-popover-border: 1px solid var(--ag-charts-border-color);
+    --ag-charts-popover-active-color: var(--ag-active-color, #5090dc);
+    --ag-charts-popover-active-background-color: #e1f0fb;
+
+    /* Buttons */
+    --ag-charts-button-radius: var(--ag-border-radius, 4px);
+
+    /* Focus Indicator */
+    --ag-charts-focus-border-color: var(--ag-input-focus-border-color, var(--ag-charts-active-color));
+    --ag-charts-focus-border: solid 1px var(--ag-charts-focus-border-color);
+    --ag-charts-focus-border-shadow: var(
+        --ag-input-focus-box-shadow,
+        0 0 0 3px color-mix(in srgb, transparent, var(--ag-input-focus-border-color, var(--ag-charts-active-color)) 47%)
+    );
+
+    /* Icons */
+    --ag-charts-icon-font-family: 'agChartsDefault';
```

Comment:
I'm unsure what all the `--ag-charts-icon-font-XXX` variables are for? 

Doesn't look like they get used anywhere, and I didn't think we had icons fonts anywhere in charts?

Reply:
Good question! These are bits I forgot to remove when I changed it from an icon font to base64 uri. Can be removed.

Reply:
Let's do the CSS cleanups in a new PR - this is just moving what's already there into a new file